shopinvader_api_delivery_carrier/security/acl_account_tax.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
shouldn't RR go to their own module?

Reply:
This was done for principal modules, but not everywhere (for eg. RR are in `shopinvader_api_sale_loyalty`).
@lmignon what's your opinion about it?

Reply:
It's fine to get it here. We try to get a proper security declaration by api.... This addon comes with its own security group. If a same alc is defined multiple time across different addon we could create a common addon but is-it really better to get one addon for one ACL?